dotCMS/src/main/java/com/dotmarketing/portal/struts/DotPortletAction.java

```diff
@@ -893,6 +878,42 @@ public String getForward(ActionRequest req, String defaultValue) {
 		}
 	}
 
+	protected static void _checkUserPermissions(String inode, int permission) throws ActionException {
+		PermissionAPI perAPI = APILocator.getPermissionAPI();
```

Comment:
rename to permissionAPI and set to final

Reply:
Done